app/src/dog/del/app/api/v1/docs/List.kt

```diff
@@ -43,9 +41,10 @@ fun Route.list(store: TransientEntityStore) = get {
                     host = origin.host
                     path(it.slug)
                 },
-                it.type.name.toLowerCase()
+                it.type.name.toLowerCase(),
+                it.stringContent ?: ""
```

Comment:
I really don't like returning the content of every document here, this should be a seperate endpoint, and will be the /documents/<id> endpoint like with hastebin. I will implement this myself real quick while doing some other small adjustments to this PR while merging.